src/sfizz/Voice.cpp

```diff
@@ -364,33 +354,28 @@ void sfz::Voice::processStereo(AudioSpan<float> buffer) noexcept
     copy<float>(rightBuffer, span1);
     add<float>(leftBuffer, rightBuffer);
     subtract<float>(span1, leftBuffer);
-    applyGain<float>(sqrtTwoInv<float>, leftBuffer);
-    applyGain<float>(sqrtTwoInv<float>, rightBuffer);
+
+    // Add const aliases to be slightly more readable
+    const auto midBuffer = leftBuffer;
+    const auto sideBuffer = rightBuffer;
```

Comment:
shouldn't these 2 be defined as sum and difference of L/R channels?
mid=L+R
side=L-R

Reply:
So the mixing happened a couple lines above but it may have been reversed. I corrected to try to be a bit clearer, does it make sense to you now?